src/components/OpenFileDialog/utils.js

```diff
@@ -2,8 +2,6 @@ import i18n from '@dhis2/d2-i18n'
 
 export const AO_TYPE_VISUALIZATION = 'visualization'
```

Comment:
`AO_TYPE_VISUALIZATION` should be used further down where it says `case 'visualization': {` (but GH didn't let me comment on that line). Same for `AO_TYPE_EVENT_VISUALIZATION`

Reply:
Fixed.